ServerCore/Areas/Identity/UserAuthorizationPolicy/AuthorizationHelper.cs

```diff
@@ -200,7 +200,7 @@ public async Task PlayerCanSeePuzzleCheck(AuthorizationHandlerContext authContex
 
             if (thisEvent != null && puzzle != null)
             {
-                if (thisEvent.AreAnswersAvailableNow)
+                if (thisEvent.AreAnswersAvailableNow || thisEvent.IsAlphaTestingEvent)
```

Comment:
What happens if a player happens to find the alpha site?

Reply:
Very good concern - basically, they get in. I'll add a reminder to set the max number of teams low (like 0) when this is set, so that admins have to create player teams (and I just enforced this for the alpha site as well).